src/event.rs

```diff
@@ -58,6 +58,15 @@ impl<'a, N: Notify + 'a> input::ActionContext for ActionContext<'a, N> {
 
     fn scroll(&mut self, scroll: Scroll) {
         self.terminal.scroll_display(scroll);
+
+        let (x, y) = (self.mouse().x, self.mouse().y);
+        let size_info = self.size_info();
+        let point = size_info.pixels_to_coords(x, y);
+        let cell_side = self.mouse().cell_side;
+        self.update_selection(Point {
+            line: point.line,
+            col: point.col
+        }, cell_side);
```

Comment:
Looking at this code, I'm questioning why we're passing line, column and cell side to the `update_selection` method. It looks like all locations where `update_selection` is called, look identical. Usually that's a good indication that it would make sense to refactor things and it shouldn't be a problem to move the code into the `update_selection` method.

Reply:
Yep, makes a lot of sense.